java/mtsj/batch/src/test/java/com/devonfw/application/mtsj/general/common/AbstractSpringBatchIntegrationTest.java

```diff
@@ -89,6 +88,7 @@ public void setup() throws Exception {
     this.flyway.importTestData(ALL_TESTS_DB_SETUP_DIR);
 
     LOG.debug("Delete tmp directory");
-    FileUtils.deleteDirectory(new File(TMP_DIR));
+    File file = new File(TMP_DIR);
+    file.delete();
```

Comment:
`File.delete()` only deletes a flat file and will IMHO fail on a directory tree.
Solution could be this:
https://www.baeldung.com/java-delete-directory#conclusion-1

Reply:
see PR https://github.com/devonfw/my-thai-star/pull/321